src/dom/dom.js

```diff
@@ -3609,7 +3610,9 @@ p5.MediaElement = class MediaElement extends p5.Element {
 
     this._prevTime = playbackTime;
   }
-};
+}
+
+p5.MediaElement = MediaElement;
```

Comment:
Might be just me but I don't quite understand what this particular syntax is doing compared to how it was before, can you perhaps explain a bit here?

Reply:
I read the p5.sound 's code, then I found that it's organized like this.

Pros: better hint for the editor. Since now all the `p5.class` becomes any, but the `class` itself has proper hints.
Cons: Need to type the name more times.

the effect is the same.